.github/workflows/build.yml

```diff
@@ -120,6 +121,18 @@ jobs:
           az storage blob download --account-name $ACCOUNT_NAME --container-name $CONTAINER_NAME --name $FILE \
             --file Bitwarden/Application/Support/$FILE --output none
 
+      - name: Retrieve beta Google Services secret
+        if: env.build-variant == 'Beta'
+        env:
+          ACCOUNT_NAME: bitwardenci
+          CONTAINER_NAME: mobile
+          SOURCE_FILE: GoogleService-Info.plist
```

Comment:
Should this be updated to `GoogleService-Info-ios-pm-beta.plist` ?

Reply:
Updated